nebula_messages/continental_msgs/msg/ContinentalArs548Object.msg

```diff
@@ -0,0 +1,44 @@
+uint32 object_id
+uint16 age
+uint8 status_measurement
+uint8 status_movement
+uint8 position_reference
+
+geometry_msgs/Point position
+geometry_msgs/Vector3 position_std
+float32 position_covariance_xy
+float32 position_orientation
+float32 position_orientation_std
+
+float32 existence_probability
+
+uint8 classification_car
+uint8 classification_truck
+uint8 classification_motorcycle
+uint8 classification_bicycle
+uint8 classification_pedestrian
+uint8 classification_animal
+uint8 classification_hazard
+uint8 classification_unknown
+
+geometry_msgs/Vector3 absolute_velocity
+geometry_msgs/Vector3 absolute_velocity_std
+float32 absolute_velocity_covariance_xy
+
+geometry_msgs/Vector3 relative_velocity
+geometry_msgs/Vector3 relative_velocity_std
+float32 relative_velocity_covariance_xy
+
+geometry_msgs/Vector3 absolute_acceleration
+geometry_msgs/Vector3 absolute_acceleration_std
+float32 absolute_acceleration_covariance_xy
+
+geometry_msgs/Vector3 relative_acceleration
+geometry_msgs/Vector3 relative_acceleration_std
+float32 relative_acceleration_covariance_xy
```

Comment:
`(optional)` The radar does not output those values for the z-axis so using `Vector3` seems a bit wasteful. Maybe implementing a `Vector2` type would be nicer?

Reply:
Sadly Vector2 does not exist in ROS (at least as far as I remember) :cry: 
That is why you end up seing the Vector2 msg defined in random packages (I don't think it would be a good fit in nebula).
In the meantime, I think Vector3 would be better than separate scalars